packages/web-components/fast-components/src/toolbar/fixtures/base.html

```diff
@@ -93,6 +97,30 @@ <h2>FAST components</h2>
         <fast-option>Option 3</fast-option>
     </fast-select>
     <fast-checkbox>Checkbox</fast-checkbox>
+    <fast-radio-group slot="end">
+        <fast-radio value="filter">Filter</fast-radio>
+        <fast-radio value="search">
+            <fast-text-field placeholder="Search"></fast-text-field>
```

Comment:
Keyboarding to the `fast-text-field` is inconsistent when nested in the `fast-radio`, but that's not relevant to the keyboard focus (which works great for me across all browsers).

Reply:
yes, this is when it is nested inside of the fast-radio. The fast-radio must be selected and then you can tab to the text-field.

I did find an issue while looking at this related to the initial tab sequence that I am looking in to.